src/test/kotlin/racingcar/domain/CarTest.kt

```diff
@@ -0,0 +1,51 @@
+package racingcar.domain
+
+import io.kotest.core.spec.style.BehaviorSpec
+import io.kotest.data.forAll
+import io.kotest.data.headers
+import io.kotest.data.row
+import io.kotest.data.table
+import io.kotest.matchers.shouldBe
+import racingcar.domain.Car.Companion.MOVE_THRESHOLD
+import racingcar.domain.Car.Companion.START_POSITION
+import racingcar.domain.Car.Companion.STEP
+import racingcar.util.NumberGenerator.Companion.MAX_GENERATED_NUMBER
+import racingcar.util.NumberGenerator.Companion.MIN_GENERATED_NUMBER
+import racingcar.util.OrderNumberGenerator
+
+class CarTest : BehaviorSpec({
+    given("movable 메소드") {
+        forAll(
+            table(
+                headers("number", "expected"),
+                *(MIN_GENERATED_NUMBER until MOVE_THRESHOLD).map { row(it, false) }.toTypedArray(),
```

Comment:
```suggestion
                *(NumberGenerator.MIN_GENERATED_NUMBER until Car.MOVE_THRESHOLD).map { row(it, false) }.toTypedArray(),
```
companion object에 정의한 상수값을 import하셔서 단순화를 잘 시켜주셨네요. 👍 
다만, import를 하게 되면, 어떤 companion object의 상수값을 사용했는지 코드만 봐서는 알기가 어렵습니다.
상수가 어디에 정의가 되어있는지 알기 쉽도록 개선해보면 어떨까요? (이하 관련 내용 동일)

Reply:
오호 듣고보니 그렇네요 ㅋㅋㅋ
https://github.com/next-step/kotlin-racingcar/pull/1142/commits/ecc4e47e678b6f1fa1e43df555c4802b8e4f3632